data/sql/updates/pending_db_world/rev_1694636334573006200.sql

```diff
@@ -0,0 +1,8 @@
+--
+ALTER TABLE `creature`
+    ADD COLUMN `Comment` VARCHAR(255) NULL DEFAULT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `CreateObject`;
```

Comment:
Is there a reason why varchar is used over text?

Reply:
Same one used by pool comments, dunno the reason :p

Reply:
ah okay. Because now you are limiting it to 255 characters.

SAI comments use TEXT